library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -29,23 +29,25 @@
 
 import rx.Observable;
 import rx.functions.Func1;
+import rx.subjects.PublishSubject;
 
 public class ListViewCard extends FrameLayout
 {
     private static final String TAG = "ListViewCard";
 
     private List<Item> mItems = new ArrayList<>();
 
+    private final PublishSubject<Item> mItemClicks = PublishSubject.create();
+    private final PublishSubject<Item> mIconClicks = PublishSubject.create();
+
     private Observable<Void> mButtonClicks;
```

Comment:
The problem is that we are exposing a null Observable. Attempting to subscribe or call operators on that Observable will cause NPEs if it hasn't been initialized yet. I usually fix that by exposing PublishSubjects. 

However, we may also be able to fix that by initializing the Observables as Observable.empty().